dbdeployer

```diff
@@ -53,10 +53,14 @@ then
   exit 1
 fi
 
-#overwrite defaults with homedir file
-if [ -f ~/.dbdeployer ]
+if [ "${allow_config_override}" = 'true' ]
```

Comment:
We are using this method all throughout dbdeployer. We can chat about it tomorrow and decide if we want an effort to change to this everywhere so it can all be tested together.